Resources/config/mongodb.xml

```diff
@@ -112,11 +111,6 @@
 
         <!-- metadata -->
         <service id="doctrine_mongodb.odm.metadata.chain" class="%doctrine_mongodb.odm.metadata.driver_chain.class%" />
-        <service id="doctrine_mongodb.odm.metadata.annotation" class="%doctrine_mongodb.odm.metadata.annotation.class%">
-            <argument type="service" id="doctrine_mongodb.odm.metadata.annotation_reader" />
-            <argument>%doctrine_mongodb.odm.document_dirs%</argument>
-        </service>
-        <service id="doctrine_mongodb.odm.metadata.annotation_reader" alias="annotation_reader" />
         <service id="doctrine_mongodb.odm.metadata.attribute" class="%doctrine_mongodb.odm.metadata.attribute.class%">
```

Comment:
I'm wondering if these services are necessary, I'll try to remove them in another PR